shenzhen-solitaire.py

```diff
@@ -81,7 +104,10 @@ def grab_screenshot():
     """Grab image of current screen."""
     click_window()
     time.sleep(.1)
-    return pyscreenshot.grab()
+    img = sct.grab(sct.monitors[2])
```

Comment:
Note: I've used `sct.monitors[2]` because that's my 'primary monitor'. I can add something that selects the first entry with `{'top':0, 'left':0}`; as is, I think it tries to organize roughly 'left-to-right, top-to-bottom' but I haven't read `mss`'s source on how exactly it happens:
```python
sct.monitors
[
    {'left': -1920, 'top': -245, 'width': 4859, 'height': 2200},    # "all monitors"
    {'left': -1920, 'top': 0, 'width': 1920, 'height': 1080},    # left monitor
    {'left': 0, 'top': 0, 'width': 1920, 'height': 1080},    # "primary"
    {'left': 1920, 'top': -245, 'width': 900, 'height': 1600},    # right monitor
    {'left': 1915, 'top': 1355, 'width': 1024, 'height': 600}    # bottom-right monitor
]
```
![image](https://user-images.githubusercontent.com/2228057/160303210-e9742bf8-8226-4973-8ede-ffb9d10e4a4f.png)